src/Resources/config/services.yml

```diff
@@ -0,0 +1,5 @@
+services:
+    contao_newsletter.data_container.member:
```

Comment:
we sure to have them, but in the old classes. That's why I said we should start writing new code to give an example, and here's my take (which I already use on my custom stuff).